nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/events/FilesDataFormatHandler.java

```diff
@@ -49,19 +50,36 @@ public boolean handlePaste(Object o, PolyChart chart) {
                 MoleculeUtils.addActiveMoleculeToCanvas();
             }
         } else {
-            Platform.runLater(() -> {
-                chart.setActiveChart();
-                boolean isDataset = NMRDataUtil.isDatasetFile(files.get(0).getAbsolutePath()) != null;
-                if (isDataset) {
-                    for (File file : files) {
-                        chart.getController().openDataset(file, true);
-                    }
-                } else {
-                    chart.getController().openFile(files.get(0).getAbsolutePath(), true, false);
-                }
-
-            });
+            Platform.runLater(() -> openDataFiles(files, chart));
         }
         return true;
     }
+
+    /**
+     * Checks if first file in list is dataset is FID or processed. If it is an FID, the first file is opened, if
+     * it is processed, all the files are opened in dataset append mode.
+     * @param files The files to open.
+     * @param chart The chart to display the files in.
+     */
+    private void openDataFiles(List<File> files, PolyChart chart) {
+        chart.setActiveChart();
+        boolean isFID = false;
+        try {
+            isFID = NMRDataUtil.getFID(files.get(0).getAbsolutePath()).isFID();
+        } catch (IOException e) {
+            // If the file can't be opened as an FID NMRData, check if it is a datafile, since processed Bruker files
+            // will not open with NMRDataUtil.getFID, otherwise return
+            if (NMRDataUtil.isDatasetFile(files.get(0).getAbsolutePath()) == null) {
+                log.warn("Unable to open datafiles. {}", e.getMessage(), e);
+                return;
+            }
```

Comment:
processed Bruker files need this extra check as they won't load as NMRData with getFID.

@brucejohnson  Note: the drag and drop works for the 1r, 1i, and 2rr files only from the processed files shared. These are also the only 3 files that can be opened using "Open Dataset"